kubernetes-server-mock/pom.xml

```diff
@@ -65,6 +65,10 @@
       <version>${slf4j.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-model-core</artifactId>
+    </dependency>
```

Comment:
I'm not sure if this is required. Doesn't `kubernetes-client` include `Kubernetes-model-core`?